src/libUtils/EvmJsonResponse.cpp

```diff
@@ -0,0 +1,182 @@
+/*
+ * Copyright (C) 2022 Zilliqa
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+#include "libUtils/EvmJsonResponse.h"
+#include "common/Constants.h"
+#include "depends/websocketpp/websocketpp/base64/base64.hpp"
+#include "nlohmann/json.hpp"  // NOLINT(readability-redundant-declaration)
+
+using websocketpp::base64_decode;
+
+namespace evmproj {
+
+evmproj::CallRespose& GetReturn(const Json::Value& oldJason,
+                                evmproj::CallRespose& fo) {
+  nlohmann::json newJason;
+
+  try {
+    newJason = nlohmann::json::parse(oldJason.toStyledString());
+  } catch (std::exception& e) {
+    LOG_GENERAL(WARNING, "Error parsing json from evmds " << e.what());
+    return fo;
```

Comment:
Wouldn't this return some uninitialized `fo`?

Reply:
well caught, addressed with :

 try {
    newJason = nlohmann::json::parse(oldJason.toStyledString());
  } catch (std::exception& e) {
    LOG_GENERAL(WARNING,
                "Exception JSONRPC parser to nlohman parser " << e.what())
    throw e;
  }